graph-oauth/src/accesstoken.rs

```diff
@@ -61,6 +62,17 @@ pub struct AccessToken {
     jwt: Option<JsonWebToken>,
 }
 
+use std::str::FromStr;
```

Comment:
Imports should go at the top of the file.

Reply:
Also if you can, run cargo clippy and cargo fmt for lints and formatting and make sure clippy doesnt fail for anything.